packages/middleware-sdk-glacier/src/configurations.ts

```diff
@@ -0,0 +1,17 @@
+export interface AddGlacierApiVersionMiddlewareInputConfig {}
+
+interface PreviouslyResolved {
+  apiVersion: string;
+}
+
+export interface ResolvedAddGlacierApiVersionMiddlewareConfig {
+  apiVersion: string;
+}
+
+export function resolveSsecMiddlewareConfig<T>(
```

Comment:
If this plugin is called `GlacierCustomizationsPlugin` these interfaces should be `GlacierCustomizationsResolvedConfig` and `GlacierCustomizationInputConfig`

Reply:
fixed in 1964189290c898e22dce6a0bd25d206bf0e20e7f